src/sentry/receivers/outbox/region.py

```diff
@@ -118,3 +118,16 @@ def maybe_join_org(org_member: OrganizationMember):
                     organization_id=org_member.organization_id,
                     user_id=org_member.user_id,
                 )
+
+
+@receiver(process_region_outbox, sender=OutboxCategory.ORGANIZATION_MAPPING_CUSTOMER_ID_UPDATE)
+def process_organization_mapping_customer_id_update(
+    object_identifier: int, payload: Any, **kwds: Any
+):
+    if (org := maybe_process_tombstone(Organization, object_identifier)) is None:
+        return
+
+    if payload and "customer_id" in payload:
+        organization_mapping_service.update(
```

Comment:
What's the behavior here if the organization by the given id does not exist (state inconsistency)?

Reply:
Goooood question. So looks like it currently just passes and silently eats the `DoesNotExist` error currently. 

This shouldn't be a huge deal in the "org mapping not created but still provisioning" case as we're planning to rectify issues where a customer ID is missing by repairing the mapping via the customer metadata, which contains org ID.